src/library/styles/components/_skip-link.scss

```diff
@@ -1,7 +1,7 @@
 .bsds-skip-link {
   position: absolute;
   top: -100%;
-  z-index: 1000;
+  z-index: 70;
```

Comment:
i think we may be able to just remove z-index entirely here since the skip link pushes the page content down